apps/credential-showcase-api-server/src/database/repositories/PersonaRepository.ts

```diff
@@ -0,0 +1,59 @@
+import { eq } from 'drizzle-orm';
+import { Service } from 'typedi';
+import { DatabaseService } from '../../services/DatabaseService';
+import { NotFoundError } from '../../errors/NotFoundError';
+import { assets, personas } from '../schema';
+import { Persona, NewPersona, RepositoryDefinition } from '../../types';
+
+@Service()
+class PersonaRepository implements RepositoryDefinition<Persona, NewPersona> {
+    constructor(private readonly databaseService: DatabaseService) {}
+
+    async create(persona: NewPersona): Promise<Persona> {
+        const [result] = await (await this.databaseService.getConnection())
```

Comment:
we need to check if the images exists before adding the persona. we already have code for this that also returns a human readable error